tests/test_blstm.py

```diff
@@ -0,0 +1,60 @@
+import onnxruntime as ort
+import tempfile
+import torch
+from torch.onnx import export as export_onnx
+
+from i6_models.parts.blstm import BlstmEncoder, BlstmEncoderConfig
+
+
+def test_blstm_onnx_export():
```

Comment:
This makes me wonder: Do we either want some example export somewhere? Or maybe even a Blueprint for copy pasting? Or do we just use this function (or part of it) as blueprint and maybe document it?

Reply:
I would say not for now, we can unify this later on.